src/ImageSharp/PixelFormats/PixelOperations{TPixel}.cs

```diff
@@ -29,10 +29,12 @@ public partial class PixelOperations<TPixel>
         /// <param name="count">The number of pixels to convert.</param>
         internal virtual void PackFromVector4(ReadOnlySpan<Vector4> sourceVectors, Span<TPixel> destinationColors, int count)
         {
-            GuardSpans(sourceVectors, nameof(sourceVectors), destinationColors, nameof(destinationColors), count);
+            ReadOnlySpan<Vector4> sourceVectors1 = sourceVectors;
```

Comment:
How come these are reassigned?

Reply:
Good catch! Result of undoing a refactor with R#. Will fix it.